### DISCUSSION TITLE
Suricata Rules Edit Problems

### DISCUSSION THREAD

**Discussion by Philomelaa** [Security-Onion-Solutions/securityonion#8379] [Unsupported Versions]
**Hi everyone,**

I want to edit Suricata rules. As shown in the video (https://www.youtube.com/watch?v=1jEkFIEUCuI), I applied the threshold values on global.sls according to myself. However, there was no change in the operation of the rule.

Also, when I added the existing rule to local.rules by applying the changes again (edit like that content:!"192.168.1.1"), there was no change in the operation of the rule.

What suggestions can you offer for the specific regulation of the rules?

**Comment by reyesj2** [comment]
Can you provide a santized global.sls? 

When you made your changes to the global.sls file did you restart suricata? Any errors when restarting? How are you verifying there was no change in how the rule is operating?

**Reply by Philomelaa** [reply]
Yes i did, i mean restart suricata but i don't get any error when restarting. I was test my changes one rule with from security onion documentation, but no change for rule.

**Comment by dougburks** [comment]
> I want to edit Suricata rules. As shown in the video (https://www.youtube.com/watch?v=1jEkFIEUCuI), I applied the threshold values on global.sls according to myself. However, there was no change in the operation of the rule.

Can you share the actual syntax you used?

> Also, when I added the existing rule to local.rules by applying the changes again (edit like that content:!"192.168.1.1"), there was no change in the operation of the rule.

When you created the local copy, did you disable the original rule?

**Comment by Philomelaa** [comment]
Of course, my Suricata rule looks like that alert ip any any -> any any (msg:"GPL ATTACK_RESPONSE id check returned root 2"; content:"uid=0|28|root|29|"; classtype:bad-unknown; sid:7000000; rev:1;)  and my suppress config from global.sls like that 

thresholding
  sids:
   7000000:
   - suppress:
       gen_id: 1
       track: by_dst
       ip: 192.168.1.1 

Thanks for support!

**Comment by dougburks** [comment]
I asked previously, but I don't see a response so I'll ask again. When you created the local copy, did you disable the original rule?

**Comment by Philomelaa** [comment]
Yes i did, when i testing but never changed.

**Comment by dougburks** [comment]
Have you checked the Suricata log in `/opt/so/log/suricata/` for additional clues?
